app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
I'd move this file into `stylesheets/styles/` then import it into `application.scss`